src/store/remove_other.rs

```diff
@@ -0,0 +1,25 @@
+use crate::processor::{ProcessingState, Processor};
+
+use crate::protocol::Event;
+use crate::types::{Annotated, Meta};
+
+pub struct RemoveOtherProcessor;
+
+impl Processor for RemoveOtherProcessor {
+    fn process_event(
```

Comment:
We should probably remove other in interfaces like `Exception` or `stacktrace`, too.

Reply:
`other` does not lead to an error except toplevel afaik.

Do we want to do this for all `other`?

Reply:
Regarding error, you are right. However, there are a couple of known legacy keys such as `applecrashreport`, which we shouldn’t error for even on top level.

Reply:
I think I fixed it